Update configure-email-forwarding.md (#953)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/email/configure-email-forwarding.md

    
- Email forwarding requires that the  *from*  account has a license. If you're setting up email forwarding because the user has left your organization, another option is to [convert their mailbox to a shared mailbox](convert-user-mailbox-to-shared-mailbox.md). This way several people can access it. However, a shared mailbox cannot exceed 50GB. 
    

> You must be an Exchange administrator or Global administrator in Office 365 to do these steps. For more information check [admin permissions](../add-users/about-admin-roles.md)

Reviewer: ```suggestion
You must be an Exchange administrator or Global administrator in Office 365 to do these steps. For more information, see the topic [About admin roles](../add-users/about-admin-roles.md).
```
